src/ccmodules/package_upgrade.c

```diff
@@ -62,7 +62,7 @@ void package_upgrade_handler(GNode *node) {
 #elif defined(PACKAGE_MANAGER_YUM)
 		exec_task("/usr/bin/yum update");
 #elif defined(PACKAGE_MANAGER_DNF)
-		exec_task("/usr/bin/dnf update --refresh");
+		exec_task("/usr/bin/tdnf update --refresh");
```

Comment:
/usr/bin/tdnf is standard path used in Photon OS, but user can easily configure the location.